registration/tests.py

```diff
@@ -369,7 +499,7 @@ def test_build_jembi_json_clinic_self(self):
             'type': 3,
             'swt': 1
         }
-        json = jembi_post_json.build_jembi_json(reg)
+        json = tasks.build_jembi_json(reg)
```

Comment:
careful using core language module names as variable names - better to use something unambiguous